diff/walking/differ.go

```diff
@@ -41,7 +41,16 @@ type walkingDiff struct {
 }
 
 var emptyDesc = ocispec.Descriptor{}
-var uncompressed = "containerd.io/uncompressed"
+
+const (
+	// LabelUncompressed is set to the digest of uncompressed layer
+	// for a compressed layer.
+	LabelUncompressed = "containerd.io/uncompressed"
+	// LabelEmptyLayer is set for an empty layer.
+	// The value of the label is set to an empty string.
```

Comment:
We can't use empty value labels, the empty value is used to delete a label

Reply:
can we set the value to "true"?

Reply:
Update PR: now the value is set to `"true"`. RFC.